libs/core/src/components/pds-toast/pds-toast.scss

```diff
@@ -0,0 +1,176 @@
+:host {
+  --animation-duration: 0.3s;
+  --animation-timing: cubic-bezier(0.4, 0, 0.2, 1);
+  --padding-inline: var(--pine-dimension-lg);
+  --padding-inline-desktop: var(--pine-dimension-2xl);
+  --sizing-height-default: 68px;
+  --sizing-min-width: calc(var(--sizing-total-width) - (var(--padding-inline) * 2));
+  --sizing-min-width-desktop: calc(var(--sizing-total-width) - (var(--padding-inline-desktop) * 2));
+  --sizing-total-width: 350px;
+
+  box-sizing: border-box;
+  display: block;
+  font: var(--pine-typography-body-medium);
+}
+
+.pds-toast {
+  align-items: center;
+  animation: pds-toast--animate-in var(--animation-duration) var(--animation-timing);
+  background-color: var(--pine-color-primary);
+  border-radius: var(--pine-dimension-xs);
+  box-shadow: var(--pine-box-shadow);
+  color: var(--pine-color-text-primary);
+  display: flex;
+  height: var(--sizing-height-default);
+  justify-content: flex-start;
+  margin-block-end: var(--pine-dimension-2xs);
+  max-width: 90vw;
+  min-width: var(--sizing-min-width);
+  padding: 0 var(--padding-inline);
```

Comment:
Looks like the padding here is 32px. Can we get it to match the 24px in the Figma file?